lighthouse-core/test/audits/metrics/total-blocking-time-test.js

```diff
@@ -6,27 +6,51 @@
 'use strict';
 
 const TBTAudit = require('../../../audits/metrics/total-blocking-time.js');
-const options = TBTAudit.defaultOptions;
+const defaultOptions = TBTAudit.defaultOptions;
 
-const pwaTrace = require('../../fixtures/traces/progressive-app-m60.json');
+const trace = require('../../fixtures/traces/progressive-app-m60.json');
+const devtoolsLog = require('../../fixtures/traces/progressive-app-m60.devtools.log.json');
 
-function generateArtifactsWithTrace(trace) {
+function generateArtifacts({trace, devtoolsLog, TestedAsMobileDevice}) {
   return {
     traces: {[TBTAudit.DEFAULT_PASS]: trace},
-    devtoolsLogs: {[TBTAudit.DEFAULT_PASS]: []},
+    devtoolsLogs: {[TBTAudit.DEFAULT_PASS]: devtoolsLog},
+    TestedAsMobileDevice,
   };
 }
+
+function generateContext({throttlingMethod}) {
+  const settings = {throttlingMethod};
+  return {options: defaultOptions, settings, computedCache: new Map()};
+}
 /* eslint-env jest */
 
 describe('Performance: total-blocking-time audit', () => {
   it('evaluates Total Blocking Time metric properly', async () => {
-    const artifacts = generateArtifactsWithTrace(pwaTrace);
-    const settings = {throttlingMethod: 'provided'};
-    const context = {options, settings, computedCache: new Map()};
-    const output = await TBTAudit.audit(artifacts, context);
+    const artifacts = generateArtifacts({trace, devtoolsLog, TestedAsMobileDevice: true});
+    const context = generateContext({throttlingMethod: 'provided'});
 
+    const output = await TBTAudit.audit(artifacts, context);
     expect(output.numericValue).toBeCloseTo(48.3, 1);
     expect(output.score).toBe(1);
     expect(output.displayValue).toBeDisplayString('50\xa0ms');
   });
+
+  it('adjusts scoring based on form factor', async () => {
+    const artifactsMobile = generateArtifacts({trace, devtoolsLog, TestedAsMobileDevice: true});
+    const contextMobile = generateContext({throttlingMethod: 'simulate'});
```

Comment:
same here

Reply:
> same here

yeah, lcp was just c/p from here and is fine to switch, but for TBT I was just trying to slow things down enough to get a scoring difference (with `provided` both are 1). I'll find some other trace to test.